<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetroPlus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/image/logo.png</normaloff>:/new/prefix1/image/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="user">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);
border-radius: 20px;
border-color: rgb(255, 255, 255);
border-width:1px 1px;
border-style:solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>用户名Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(31, 212, 175, 255), stop:1 rgba(0, 204, 102, 255));
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>登录Log in</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/LOGO2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);
border-radius: 20px;
border-color: rgb(255, 255, 255);
border-width:1px 1px;
border-style:solid;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 10pt &quot;微软雅黑&quot;;
background:transparent;
border-width:0;
border-style:outset;
color: rgb(121, 121, 121);
text-align : left;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="forget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background:transparent;
border-width:0;
border-style:outset;
color: rgb(121, 121, 121);
text-align : right;</string>
    </property>
    <property name="text">
     <string>忘记密码？</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_signup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);
border-radius: 20px;
border-color: rgb(255, 255, 255);
border-width:1px 1px;
border-style:solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>用户名Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_signup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);
border-radius: 20px;
border-color: rgb(255, 255, 255);
border-width:1px 1px;
border-style:solid;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_signup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);
border-radius: 20px;
border-color: rgb(255, 255, 255);
border-width:1px 1px;
border-style:solid;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>邮箱Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_ready">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(31, 212, 175, 255), stop:1 rgba(0, 204, 102, 255));
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>注册Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="signup_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="signup_title">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: 600 20pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>注册Sign Up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_signup_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(240, 240, 240);
border-radius: 20px;
border-color: rgb(255, 255, 255);
border-width:1px 1px;
border-style:solid;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>再次输入密码Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signuptologin">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background:transparent;
border-width:0;
border-style:outset;
color: rgb(121, 121, 121);
text-align : right;</string>
    </property>
    <property name="text">
     <string>已有帐号？去登录Log in!</string>
    </property>
   </widget>
   <zorder>login</zorder>
   <zorder>email_signup</zorder>
   <zorder>signup_ready</zorder>
   <zorder>signup_label</zorder>
   <zorder>user</zorder>
   <zorder>logo</zorder>
   <zorder>password</zorder>
   <zorder>signup</zorder>
   <zorder>forget</zorder>
   <zorder>user_signup</zorder>
   <zorder>password_signup</zorder>
   <zorder>signup_title</zorder>
   <zorder>password_signup_2</zorder>
   <zorder>signuptologin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
